tests/agent/test_session_persister.py

```
@@ -0,0 +1,939 @@
+
+@pytest.fixture
+def persister(logs_dir, mock_session_db):
+    """SessionPersister with a mock DB and a temp logs_dir."""
+    return SessionPersister(
+        session_id="test_session_001",
+        session_db=mock_session_db,
+        logs_dir=logs_dir,
+        model="anthropic/claude-opus-4.6",
+        base_url="https://openrouter.ai/api/v1",
+        platform="cli",
+        session_start=datetime(2025, 6, 1, 12, 0, 0),
```

Ruff (0.15.2)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)